python/lsst/daf/butler/core/ddl.py

```diff
@@ -125,14 +143,55 @@ def process_result_value(self, value, dialect):
         return ConvexPolygon.decode(super().process_result_value(value, dialect))
 
 
+class TimeRangeWarning(Warning):
```

Comment:
Is this used now?

Reply:
Hmm, I though I cleaned it, will try again 🙂